### DISCUSSION TITLE
can't login behind reverse [HA]proxy - (login form still wants to connect over http)

### DISCUSSION THREAD

**Discussion by killtux** [firefly-iii/firefly-iii#6346] [Firefly III]
hello guys. i set up firefly over docker and it runs without any issues local. i set an reverse proxy (HAproxy) before it to access it from mobile but somehow i cant login.

![image](https://user-images.githubusercontent.com/36768145/184817273-ff9e1c67-270b-45f6-a336-2f02f2b1391f.png)

did i miss something?

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/faq/other/#i-cant-seem-to-get-https-working-with-caddy

**Reply by killtux** [reply]
> https://docs.firefly-iii.org/firefly-iii/faq/other/#i-cant-seem-to-get-https-working-with-caddy

trusted proxies is already "**". any other suggestions?

**Reply by JC5** [reply]
Not sure if it's relevant, but make sure the right headers are set so Firefly III knows it's supposed to return https:

https://github.com/orgs/firefly-iii/discussions/4814#discussioncomment-801483

**Reply by killtux** [reply]
thanks for the suggestion. since i am using HAproxy, adding 
> http-request set-header X-Forwarded-Proto https

to the backend pool config worked ☺

✔ **https://github.com/firefly-iii/firefly-iii/issues/1062#issuecomment-353739908**
